tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/service/ProcessService.java

```diff
@@ -83,53 +80,42 @@ public ProcessInstanceDTO startProcessInstance(
       final String processDefinitionKey,
       final List<VariableInputDTO> variables,
       final String tenantId) {
+    return startProcessInstance(processDefinitionKey, variables, tenantId, true);
+  }
 
-    if (!identityAuthorizationService.isAllowedToStartProcess(processDefinitionKey)) {
-      throw new ForbiddenActionException(
-          "User does not have the permission to start this process.");
-    }
+  public ProcessInstanceDTO startProcessInstance(
+      final String processDefinitionKey,
+      final List<VariableInputDTO> variables,
+      final String tenantId,
+      final boolean executeWithAuthentication) {
 
     final boolean isMultiTenancyEnabled = tenantService.isMultiTenancyEnabled();
 
     if (isMultiTenancyEnabled && !tenantService.getAuthenticatedTenants().contains(tenantId)) {
       throw new InvalidRequestException("Invalid tenant.");
     }
```

Comment:
❓ If no one is authenticated, how will we get the authenticated tenants?

Reply:
If no authentication is provided, then no tenant check will happen. This is handled in the tenant service.